README.md

```diff
@@ -477,13 +477,10 @@ In this case, we can just change the override to point to some commit of our for
 
 
 ```haskell
-let overrides =
-    { simple-json =
-          upstream.simple-json
-       // { repo = "https://github.com/my-user/purescript-simple-json.git"
-          , version = "701f3e44aafb1a6459281714858fadf2c4c2a977"
-          }
-    }
+let upstream = --
```

Comment:
I kind of like leaving out the URL and hoping that people don't try to copypaste this, but I'm wondering if we can find a way to make this self contained? :thinking:

Reply:
I could clarify that via a more detailed comment
```dhall
let upstream = -- put package set release URL here
```
When people do `spago init`, the file itself will indicate how that works.

Reply:
How does latest commit's changes look?

Reply:
Looks great, cheers :clap: